### ISSUE TITLE
Viewing Response Body Icon

### ISSUE THREAD

**Issue by dhit-mdavis** [onc-healthit/onc-certification-g10-test-kit#502]
Hi all. 

Much appreciated on adding the copy button and the drop down icon for viewing the Response Body of a FHIR Request.

I have a little feedback that you can take as is or apply however you see fit.

1. While having the copy button and drop down icon is great, on 1st experience, I thought maybe something was wrong or broken
2. There's not a clear enough indication that the JSON is collapsed by default and that you need to click the drop down icon to expand
3. Maybe adding some background color in a circle or adding some animation for the drop down icon to go up and down may help

![image](https://github.com/onc-healthit/onc-certification-g10-test-kit/assets/56314435/6cf3bfd8-eb2a-48c6-bf0b-87eb96b5d4af)

Other than that, little usability fixes like you all have been adding are great for those who use this wonderful tool day in and day out.

Thanks - Mat

**Comment by arscan** [comment]
Thank you very much for your feedback and kind words about Inferno.  We'll see if we can make that a little more obvious in future versions.

**Comment by dhit-mdavis** [comment]
No problem at all Rob. Thank y'all as always!

**Comment by AlyssaWang** [comment]
Hi Mat, thanks for the feedback! We've made some updates to the way the Response Body looks to help alleviate these issues. The icons should now be larger and more visible, and we've added color and interactions to the body header to make the body easier to access. We've also added a line count to the header title to suggest that the body is collapsed. 

I'm going to go ahead and close this issue for now, but definitely reach out if you feel it should be reopened or there are other issues to report. Thanks again!
